Model/Export/Orders.php

```diff
@@ -549,4 +539,32 @@ protected function getOrderPostCode(\Magento\Sales\Api\Data\OrderInterface $orde
 
         return $postCode;
     }
+
+    protected function getOrders($storeId, $fromDate, $toDate){
+        $orderList = $this->orderCollection->create();
+
+        $select = $orderList->getSelect();
+        $select->joinLeft(
+            ["shipment" => "sales_shipment"],
+            'main_table.entity_id = shipment.order_id',
+            []
+        )->joinLeft(
+            ["shipment_track" => "sales_shipment_track"],
+            'shipment.entity_id = shipment_track.parent_id',
+            ['ship_updated_at' => 'shipment_track.updated_at']
+        );
+
+        $orderList->addFieldToFilter(self::MAIN_TABLE_PREFIX . self::STORE_ID_FIELD_ID, ['eq' => $storeId]);
+        $orderList->addFieldToFilter(
+            [self::MAIN_TABLE_PREFIX . self::UPDATED_AT_FIELD_ID, 'shipment_track.updated_at'], [
+                ['gteq' => $fromDate->format(DATE_ATOM)],
+                ['gteq' => $fromDate->format(DATE_ATOM)]
+            ]
+        );
+        $orderList->addFieldToFilter(
```

Comment:
Does adding a new "addFieldToFilter" call add the query as an "AND" or and "OR"? I see the filter for the store ID on line 557, followed by the shipment "updated at" filter on the next line. I would imagine these are added as "AND" cases. However, we would want to the shipment "updated at" and order "updated at" to be OR'd together.

Reply:
Using addFieldToFilter with an array like line 558 will create an 'or' with all the conditions passed in.  Using it on a single filed like line 557 will create an 'and'. 

The query generated here is:
`SELECT `main_table`.*, `shipment_track`.`updated_at` FROM `sales_order` AS `main_table`
 LEFT JOIN `sales_shipment` AS `shipment` ON main_table.entity_id = shipment.order_id
 LEFT JOIN `sales_shipment_track` AS `shipment_track` ON shipment.entity_id = shipment_track.parent_id WHERE (`main_table`.`store_id` = '1') AND ((`main_table`.`updated_at` >= '2020-08-30T21:11:42+00:00') OR (`shipment_track`.`updated_at` >= '2020-08-30T21:11:42+00:00')) AND (`main_table`.`updated_at` <= '2020-09-01T21:11:42+00:00')`